azkaban-common/src/main/java/azkaban/Constants.java

```diff
@@ -86,6 +86,14 @@
 
     public static final String IS_METRICS_ENABLED = "azkaban.is.metrics.enabled";
 
+    // following parameters are used when there is reverse proxy between azkaban server and user, set this parameters to the valies that enduser sees
```

Comment:
"User facing web server configurations used to construct the user facing server URLs. They are useful when there is a reverse proxy between Azkaban web servers and users."?

typo: values

I use https://www.grammarly.com/ and intellij to check my spelling.